repose-aggregator/components/filters/translation/pom.xml

```diff
@@ -87,11 +86,6 @@
             <artifactId>xalan</artifactId>
             <version>2.7.1</version>
         </dependency>
-        <dependency>
```

Comment:
ahh, i see it was in the same pom twice. well that's a totally legit removal then.